src/tutorials/lua_basics_lightbulb.md

```diff
@@ -25,103 +25,184 @@ This involves turning on and off a light switch to illuminate a light bulb.
+![Trigger in Core Content](../img/LightBulb/trigger.png){: .image-frame .center loading="lazy" }
+
+1. In the Hierarchy window, select the "Trigger" object within the template and press ++F++. This will move the Main Viewport view to focus on the trigger.
+2. Notice the size, shape, and position of the trigger. The player will have to stand inside the box to be able to activate the trigger.
+
+!["Trigger Location"](../img/LightBulb/image20.png){: .image-frame .center loading="lazy" }
+
+!!! hint
+    If you can't see a the trigger, press ++V++ to make it visible!
+
+### Make the Trigger Interactable
 
-## Writing the Code
+!["Interactable"](../img/LightBulb/image12.png){: .image-frame .center loading="lazy" }
 
-### Define the Switch
+1. Look at the **Properties** window with the trigger selected. Under the **Gameplay** section there is a property called **Interactable**. Make sure the box is checked so players can interact with the trigger.
+2. Find the **Interaction Label** property, and change it to something like "Turn On"
+3. Press **Play** to start a preview, and walk up to the Light switch. You should see a message pop up in the middle of your screen.
+
+## Setting the Custom Properties
+
+A very powerful feature of creating in Core is custom properties; they allow you to point reference to other objects or other data types, such as strings, numbers, etc.
+
+Thinking ahead before we begin scripting, we need to decide what custom properties we need.
+
+Being this is a lightbulb and switch, we know those are two custom properties we'll need already.
+
+Additionally, in order to be able to allow the player's interaction to cause the switch to flip and the light to turn on, we'll need a reference to the trigger we just customized above.
+
+To add a custom property to the script, click on the script and then drag-and-drop the object we are referencing into the **Custom** section of the **Properties** window.
+
+It should now look like this:
+!["Custom Properties"](../img/LightBulb/CustomProperties.png){: .image-frame .center loading="lazy" }
+
+## Scripting References to Custom Properties
+
+### Reference the Switch
 
 We want our light switch to function just like a real one: the switch will point up or down depending on whether the light is turned on or off.
 
 First you will need to tell the script which object in the scene is the switch, so that it knows what to rotate. You will create a variable that defines what the switch is.
 
 To move the switch, we need a way to talk about the specific object from the game space in code.
 
-Type the following into Line 1 of `LightToggleScript`:
+Type the following into Line 1 of `LightSwitchToggle`:
+
+```lua
+local Switch = script:GetCustomProperty("Switch"):WaitForObject()
+```
+
+- `local` is a **keyword** that indicates that you are making a variable. We use the term `local` here because it will only be accessible from this script rather than being accessible from external scripts, or globally.
```

Comment:
cut "globally", since we don't explain it here. Consider changing "external" to "other" but I could go either way on that.